docs/updates.thunderbird.net.md

```diff
@@ -1,57 +1,344 @@
+
+```py
+'updates.128.appeal.dec24.donate': '/thunderbird/128.0/dec24/donate/',
+```
+
+***Make sure to restart the build tool after editing the `settings.py` file.***
+
+#### Call the macro
+
+Finally, call the macro where you'd like to display it:
+
+```jinja
+<section id="donate-button-container">
+  {{ donate_button(form_id=fru_form_id, campaign=utm_campaign, base_url=donation_base_url) }}
+</section>
+```
+
+After restarting the build tool and refreshing your browser, you should see the donation button on the page.
+
+### Localization
+
+All text in the Jinja template should be translated into the user's local language.
+
+#### Specifying translatable strings
+
+There are two ways to specify that a string should be translated.
+You can wrap the string in one of the following:
+
+- The `_()` function is used for shorter pieces of text.
+- A `{% trans %}` block wraps longer strings.
+
+Use `_()` for simple strings:
+
+ ```jinja
+ <p class="closing-text">{{ _('The Thunderbird Team') }}</p>
+ ```
+
+HTML tags are allowed inside a translatable string:
+
+```jinja
+<h1>
+    {{ _('Help Keep <span>Thunderbird Alive</span>') }}
+</h1>
+```
+When possible, avoid putting HTML tags inside a translatable string. One of our volunteer translators may accidentally change the tags, causing an error when building the site.
+
+`aria-label` (and `alt` text) should be translated:
+
+```jinja
+<h1 id="appeal-heading" aria-label="{{ _('Help Keep Thunderbird Alive!') }}">
+  {{ _('Help Keep <span>Thunderbird Alive</span>') }}
+</h1>
+```
+
+The December 2024 appeal does not use `{% trans %}` blocks, but here is an example from the main `index.html`:
+
+```jinja
+{% trans %}
+  Meet Thunderbird, the <strong>email and productivity</strong> app that maximizes your freedoms.
+{% endtrans %}
+```
+
+#### Extracting strings for translation
+
+Follow the instructions located in [l10n_tools/readme.md](../l10n_tools/readme.md) to compile and extract strings.
+
+### Accessibility
+
+Provide a label using the `aria-label` for interactive elements that assistive software may not understand (such as a close button).
+
+This is an example from the November 2024 appeal page:
+
+```jinja
+<a id="donate-footer"
+   class="btn btn-no-bg"
+   aria-label="{{ _('Donate') }}"
+   href="{{ donate_url(content='bottom_cta', campaign=utm_campaign, form_id=fru_form_id, base_url=donation_base_url, source=utm_source) }}">
+  <span aria-hidden="true" class="heart-svg">{{ svg('donate-heart') }}</span>
+  {{ _('Donate') }}
+</a>
+```
+
+
+Use `aria-hidden="true"` for purely decorative content that should be ignored by assistive software:
+
+```jinja
+<aside id="illustration" aria-hidden="true">
+  <div id="roc">
+    {{ high_res_img('thunderbird/appeal/dec24/forest-roc.png', {'alt': _('')}, alt_formats=('webp', 'avif')) }}
+  </div>
+</aside>
+```
+
+Add `alt` text to images, except when the information is redundant or the image is purely decorative.
```

Comment:
In these cases it turns out to be better to just leave alt empty, and the screen reader will skip it automatically. (I learned this recently.)

Reply:
Just so I'm clear, you're suggesting that this should be something like this?

```
Make sure to add an `alt` attribute to every image tag. 

For images that are purely decorative, or when the information is redundant, use `alt=""`. Screen readers will automatically skip them.
```

Reply:
Yeppers!

Reply:
Thanks! I went with an abbreviated version: https://github.com/thunderbird/thunderbird-website/pull/724/commits/720c20e38d5e7efed1848fee64f22f1f81909d1a